router-tests/modules/module_test.go

```diff
@@ -38,3 +39,103 @@ func TestModuleSetCustomHeader(t *testing.T) {
 		assert.JSONEq(t, res.Body, `{"data":{"employees":[{"id":1},{"id":2},{"id":3},{"id":4},{"id":5},{"id":7},{"id":8},{"id":10},{"id":11},{"id":12}]}}`)
 	})
 }
+
+func TestSortingModulesByPriority(t *testing.T) {
+	cfg := config.Config{
+		Modules: map[string]interface{}{
+			"myModule": module.MyModule{
+				Value: 1,
+			},
+		},
+	}
+
+	modulesW := make(map[string]core.ModuleInfo)
+	modulesW["module1"] = core.ModuleInfo{
```

Comment:
Could you suffix the module names with the priority? This would be easy to read.

Reply:
Done